sklearn/metrics/pairwise.py

```diff
@@ -302,7 +302,7 @@ def _euclidean_distances_upcast(X, XX=None, Y=None, YY=None):
     y_density = Y.nnz / np.prod(Y.shape) if issparse(Y) else 1
 
     # Allow 10% more memory than X, Y and the distance matrix take (at least
-    # 10Mib)
+    # 10MiB)
```

Comment:
Well, 10MiB is 80Mib. Wouldn't you have to change the calculations below as well? :)

Reply:
I don't think so, I wrote Mib but it was actually MiB. MiB is 1024 * 1024 bytes, right ? If so, that's what I used